oauth2client/_helpers.py

```diff
@@ -39,15 +39,42 @@ def _json_encode(data):
   return json.dumps(data, separators=(',', ':'))
 
 
+def _to_bytes(value, encoding='ascii'):
```

Comment:
Since (elsewhere in this commit) _to_bytes is used outside this module, make it module-public by naming it "to_bytes". I think this may break a pattern otherwised used in this codebase but I think it's the right thing and it's worth it. The function won't be public-to-users-of-the-oauth2client-library because _helpers is itself a library-private module.